.github/actions/ci-setup/action.yaml

```diff
@@ -14,8 +14,12 @@ runs:
       with:
         path: ~/.local/share/pnpm/store/v4
         key: ${{ runner.os }}-node-${{ inputs.node-version }}-pnpm-${{ inputs.pnpm-version }}-lock-${{ hashFiles('**/pnpm-lock.yaml') }}
-        restore-keys: |
-          ${{ runner.os }}-pnpm
```

Comment:
Removed the `restore-key`. It was incorrect anyway as the above restore key stars with ${{ runner.os }}-node so was redundant. And we should not be restoring the cache from a different run, if versions have changed.